.github/workflows/workflow.yaml

```diff
@@ -216,7 +201,7 @@ jobs:
           git config --global --add 'url.https://github.com/.insteadOf' 'git@github.com:'
           cat ~/.gitconfig
```

Comment:
Skal denne være med her? ser at den ikke settes til noe output el.

Reply:
Det er for å forenkle debugging, tenker jeg. Den har ingen secrets i seg, men viser hva steget har gjort av konfigurasjon.